lib/winston/rejection-handler.js

```diff
@@ -88,7 +88,7 @@ module.exports = class RejectionHandler {
         err && err.stack || '  No stack trace'
       ].join('\n'),
       stack: err && err.stack,
-      exception: true,
+      rejection: true,
```

Comment:
This should have been advertised as a breaking change in the changelog / release notes. (Or for backwards compatibility, used both properties).